.env

```
@@ -16,6 +16,6 @@ OTTERSCAN_IMAGE=otterscan/otterscan:latest
 RIF_RELAY_IMAGE=boltz/rif-relay:latest
 NGINX_IMAGE=nginx:alpine
 NBXPLORER_IMAGE=nicolasdorier/nbxplorer:2.5.30
-ARKD_WALLET_IMAGE=ghcr.io/arkade-os/arkd-wallet:v0.8.10
-ARKD_IMAGE=ghcr.io/arkade-os/arkd:v0.8.10
+ARKD_WALLET_IMAGE=ghcr.io/arkade-os/arkd-wallet:v0.8.11
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ARKD_WALLET_IMAGE key should go before the BITCOIN_CORE_IMAGE key

(UnorderedKey)